setup.cfg

```diff
@@ -1,6 +1,54 @@
 [flake8]
 max-line-length = 99
-
 ignore = 
     W504,  # W504: Line break occurred after a binary operator
-    E741  # E741: Do not use variables named 'I', 'O', or 'l'
\ No newline at end of file
+    E741  # E741: Do not use variables named 'I', 'O', or 'l'
+exclude = setup.py,
+        asv/benchmarks,
+        asv/commands,
+        asv/console.py,
+        asv/environment.py,
+        asv/machine.py,
+        asv/graph.py,
+        asv/.pytest_cache/d/asv-wheels/cache/tmp
```

Comment:
Is this happening in the CI or locally? I think the CI shouldn't fail, but maybe a good idea anyway to ignore anything in `.pytest_cache` directories, since those are temporary. Not sure exactly how to ignore all directories with that name, but I'm sure you'll find out.